tests/integration_tests/test_shell_comp_write_bdf.py

```diff
@@ -44,8 +44,8 @@ def setup_tacs_problems(self, comm):
 
         # Overwrite default check values
         if self.dtype == complex:
-            self.rtol = 1e-8
-            self.atol = 1e-8
+            self.rtol = 1e-7
+            self.atol = 1e-7
             self.dh = 1e-50
         else:
             self.rtol = 2e-1
```

Comment:
Can we make these tolerances tighter?

Reply:
Unfortunately, not. For whatever reason, the accuracy tops out a little above 1e-8. I think we should take this accuracy for now

Reply:
Sorry, this comment got applied to the wrong lines, I meant the real test values

Reply:
Ah gotcha, yes we I can probably drop that a bit